### PR TITLE
Fix launch script to deploy RHOBS on any OCP cluster

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR contains some fixes around deploying RHOBS on any OCP cluster.
Related Issue: https://github.com/rhobs/configuration/issues/378

### CODE REVIEW

File: resources/services/observatorium-metrics-template.yaml [hunk 1]
```diff
@@ -4577,7 +4577,7 @@ parameters:
 - name: THANOS_CONFIG_SECRET
   value: thanos-objectstorage
 - name: THANOS_IMAGE_TAG
-  value: v0.23.1
+  value: v0.29.0
```
Comment (PhilipGough) [inline resources/services/observatorium-metrics-template.yaml:4580]: This can go to v0.30.2 already :)

**Review by douglascamata — COMMENTED** [review]
Niiiice! Thanks, @vprashar2929. 

Besides @PhilipGough's comment, LGTM.

I would like to have other team members review it too, as I don't have much experience/knowledge in this launch script.

File: resources/services/observatorium-metrics-template.yaml (hunk 1 above)
Comment (vprashar2929) [reply resources/services/observatorium-metrics-template.yaml]: Done Philip

**Review by saswatamcode — APPROVED** [review]
Thanks for the awesome work! I'd love to see this integrated with our CI too! 🙂

**Comment by vprashar2929** [comment]
> Thanks for the awesome work! I'd love to see this integrated with our CI too! 🙂

Definitely my next PR will be around that😉

**Review by PhilipGough — APPROVED** [review]

**Pull request merged**